pynamodb/connection/base.pyi

```diff
@@ -22,13 +25,21 @@ class Connection:
     host: Any
     region: Any
     session_cls: Any
-    def __init__(self, region: Optional[Any] = ..., host: Optional[Any] = ..., session_cls: Optional[Any] = ..., request_timeout_seconds: Optional[Any] = ..., max_retry_attempts: Optional[Any] = ..., base_backoff_ms: Optional[Any] = ...) -> None: ...
+    def __init__(
+        self,
+        region: Optional[Any] = ...,
+        host: Optional[Any] = ...,
+        connect_timeout_seconds: Optional[float] = ...,
+        read_timeout_seconds: Optional[float] = ...,
+        max_retry_attempts: Optional[int] = ...,
+        base_backoff_ms: Optional[int] = ...,
+        max_pool_connections: Optional[int] = ...,
+        extra_headers: Optional[MutableMapping[Text, Text]] = ...,
+    ) -> None: ...
     def dispatch(self, operation_name, operation_kwargs): ...
     @property
     def session(self): ...
```

Comment:
curious, this is not typed? hard to understand what "session" means

Reply:
Added a type for it